src/samples/server/Elsa.Samples.Server.Host/README.md

```diff
@@ -119,6 +119,16 @@ YesSql Postgre
     ...
 ```
 
+### Mandotory features
+
+"Elsa:Features:DefaultPersistence" is a mandotory feature resposible for Elsa Core persistence and must be enabled
+"Elsa:Features:WorkflowSettings" is a mandotory feature resposible for the Workflow Blueprint Settings persistence functionality and must be enabled
+
+### Workflow Settings for programmatical workflows are stored in Environment Variables and readonly, they can be set in the format as follows:
```

Comment:
This statement needs to be revised such that it makes clear that workflow settings are provided by `IWorkflowSettingsProvider` (a new type to be introduced). There would initially be two implementations as outlined earlier (DB-based and configuration-based).

Reply:
I remember we discussed it but actually we have to use both providers as they are not self sufficient providers. We just give the high priority to DB-based provider and if no settings found there we use Environment Variables. Please correct me if I am wrong.

Reply:
You're right. The DB settings should override the settings from Environment Variables. But this could work similar to the way the .NET Configuration system works, which also has multiple providers. In case of duplicate configuration keys, the last one wins. Which means that the order in which the providers are registered will matter. I think that's fine.